ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -32,7 +32,15 @@ interface {{ ethernet_interface }}
 {%         if ethernet_interfaces[ethernet_interface].lacp_timer.multiplier is arista.avd.defined %}
    lacp timer multiplier {{ ethernet_interfaces[ethernet_interface].lacp_timer.multiplier }}
 {%         endif %}
+{%     endif %}
+{%     if ethernet_interfaces[ethernet_interface].channel_group.id is arista.avd.defined and ethernet_interfaces[ethernet_interface].channel_group.mode is arista.avd.defined %}
+{%         if port_channel_interfaces['Port-Channel' ~ ethernet_interfaces[ethernet_interface].channel_group.id].lacp_fallback_mode is arista.avd.defined("individual") %}
+{%             set print_ethernet = true %}
```

Comment:
@xaviramon - I really like this approach, but I believe it is still too much abstraction for`eos_cli_config_gen`.
IMO, we should remove the conditional for port-channel, and do this type of logic in `eos_designs` instead.

Let's discuss with @ClausHolbechArista before we do any changes.

Reply:
In my opinion, roles should be independent, unless we need to combine them to make the EVPN AA functionality work together.
For this, i believe, we should support this in eos_config_cli

Reply:
I have tested a few other workarounds, but because of all the logic we already have inside `ethernet_interfaces.j2` they all break one or the other way. I would have preferred a logic-free template, so what you set is what you get. But getting there would be a breaking change.
So if we accept to have this logic in here - to avoid breaking changes - we are down to deciding how the logic should be implemented.
Line 35 and 36 can be removed since the `if` block ended by line 35 contains the same logic as line 36. The rest is fine for me.